genesyscloud/telephony_providers_edges_phonebasesettings/genesyscloud_telephony_providers_edges_phonebasesettings_proxy.go

```diff
@@ -0,0 +1,130 @@
+package telephony_providers_edges_phonebasesettings
+
+import (
+	"context"
+
+	"github.com/mypurecloud/platform-client-sdk-go/v119/platformclientv2"
+)
+
+var internalProxy *phoneBaseProxy
+
+type getPhoneBaseSettingFunc func(ctx context.Context, p *phoneBaseProxy, phoneBaseSettingsId string) (*platformclientv2.Phonebase, *platformclientv2.APIResponse, error)
+type deletePhoneBaseSettingFunc func(ctx context.Context, p *phoneBaseProxy, phoneBaseSettingsId string) (*platformclientv2.APIResponse, error)
+type putPhoneBaseSettingFunc func(ctx context.Context, p *phoneBaseProxy, phoneBaseSettingsId string, body platformclientv2.Phonebase) (*platformclientv2.Phonebase, *platformclientv2.APIResponse, error)
+type postPhoneBaseSettingFunc func(ctx context.Context, p *phoneBaseProxy, body platformclientv2.Phonebase) (*platformclientv2.Phonebase, *platformclientv2.APIResponse, error)
+type getAllPhoneBaseSettingsFunc func(ctx context.Context, p *phoneBaseProxy) (*[]platformclientv2.Phonebase, error)
+
+// phoneBaseProxy contains all of the methods that call genesys cloud APIs.
+type phoneBaseProxy struct {
+	clientConfig *platformclientv2.Configuration
+	edgesApi     *platformclientv2.TelephonyProvidersEdgeApi
+
+	getPhoneBaseSettingAttr     getPhoneBaseSettingFunc
+	deletePhoneBaseSettingAttr  deletePhoneBaseSettingFunc
+	putPhoneBaseSettingAttr     putPhoneBaseSettingFunc
+	postPhoneBaseSettingAttr    postPhoneBaseSettingFunc
+	getAllPhoneBaseSettingsAttr getAllPhoneBaseSettingsFunc
+}
+
+// newPhoneBaseProxy initializes the Phone Base proxy with all of the data needed to communicate with Genesys Cloud
```

Comment:
add in "Phone Base Setting" to make it specific

Reply:
addressed